sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -2571,12 +2605,61 @@ __ESIMD_API simd<Tx, N> slm_atomic_update(simd<uint32_t, N> offsets,
 
 /// @} sycl_esimd_memory_slm
 
+namespace detail {
+
+/// USM pointer atomic.
+/// Supported platforms: DG2, PVC
+/// VISA instruction: lsc_atomic_<OP>.ugm
+///
+/// @tparam Op is operation type.
+/// @tparam T is element type.
+/// @tparam N is the number of channels (platform dependent).
+/// @tparam L1H is L1 cache hint.
+/// @tparam L3H is L3 cache hint.
```

Comment:
Same nit

Reply:
Done in https://github.com/intel/llvm/pull/11833/commits/ffe19702c45355afd8b7ab17d2170cd783e9f670